test/MINLPTests/run_minlptests.jl

```diff
@@ -98,7 +101,26 @@ CONFIG["Ipopt"] = Dict(
 #     "infeasible_point" => AmplNLWriter.MOI.UNKNOWN_RESULT_STATUS,
 # )
 
-@testset "$(name)" for name in ["Ipopt", "Bonmin", "Couenne"]
+import Uno_jll
+
+CONFIG["Uno"] = Dict(
+    "mixed-integer" => false,
+    "amplexe" => Uno_jll.amplexe,
+    "options" => String["preset=ipopt", "linear_solver=MUMPS"],
+    "tol" => 1e-5,
+    "dual_tol" => 1e-5,
+    "nlp_exclude" => String[],
+    "nlpcvx_exclude" => String[],
+    "nlpmi_exclude" => String[],
+    "infeasible_point" => AmplNLWriter.MOI.NO_SOLUTION,
+)
+
+# Uno needs the full uno.options file...
+run(
+    `curl https://raw.githubusercontent.com/cvanaret/Uno/refs/heads/main/uno.options -o uno.options`,
+)
```

Comment:
Needs https://github.com/cvanaret/Uno/issues/28

Reply:
Default options are now hard coded, which means that Uno can run without an option file. They can be overwritten by command-line options/a preset/a partial option file. See cvanaret/Uno#28

Reply:
Requires https://github.com/JuliaPackaging/Yggdrasil/pull/9652

Reply:
Requires https://github.com/JuliaRegistries/General/pull/117704